mathesar_ui/src/component-library/labeled-input/LabeledInput.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import Label from '@mathesar-component-library-dir/label/Label.svelte';
+  import type { LabeledInputLayout } from './LabeledInput.d';
+  
+  export let label: string | undefined = undefined;
+  export let layout: LabeledInputLayout = 'inline';
+</script>
+
+<div
+  class="labeled-input"
+  class:layout-stacked={layout === 'stacked'}
+  class:layout-inline={layout === 'inline'}
+  class:layout-inline-input-first={layout === 'inline-input-first'}
```

Comment:
We could probably replace this with:

```suggestion
  class="labeled-input layout-{layout}"
```